packages/lib/googleSheet/service.ts

```diff
@@ -31,8 +37,12 @@ async function fetchSpreadsheets(auth: any) {
   }
 }
 
-export const getGoogleSheetIntegration = cache(
-  async (environmentId: string): Promise<TGoogleSheetIntegration | null> => {
+export const getGoogleSheetIntegration = (
```

Comment:
Not sure about this function. I think it makes sense that there is a googleSheets service for the interaction with Google API, but `getGoogleSheetIntegration` is essentially a `getIntegrationByType` function for the integration service.

Reply:
are you saying to refactor this one to be more generic rather than specific to Google Spreadsheet like `getIntegrationByType`?